ssl/quic/json_enc.c

```diff
@@ -603,19 +602,16 @@ void ossl_json_f64(OSSL_JSON_ENC *json, double value)
     if (!json_pre_item(json))
         return;
 
-#if defined(__STDC_VERSION__) && __STDC_VERSION__ >= 199901L
     {
-        int checks = isnan(value);
-# if !defined(OPENSSL_SYS_VMS)
-        checks |= isinf(value);
-# endif
+        /* Figure out if we've been passed a NaN or infinty */
+        const double x = 0 * value;     /* NaN if value is NaN or infinity */
```

Comment:
0 otherwise?

Reply:
It can be either 0 or -0.  Since we only care about NaN, does it need a comment about zeros?